frontend/src/metabase/components/DriverWarning/DriverWarning.jsx

```diff
@@ -8,51 +8,70 @@ import {
   engineSupersedesMap,
 } from "metabase/entities/databases/forms";
 
-import Warnings from "metabase/query_builder/components/Warnings";
-
 import {
   CardContent,
   DriverWarningContainer,
   IconContainer,
+  WarningIcon,
+  WarningParagraph,
 } from "./DriverWarning.styled";
+
 import ExternalLink from "metabase/components/ExternalLink";
+import Icon from "metabase/components/Icon";
 import MetabaseSettings from "metabase/lib/settings";
 
 const propTypes = {
   engine: PropTypes.string.isRequired,
+  hasCircle: PropTypes.bool.isRequired,
+  onChangeEngine: PropTypes.func.isRequired,
 };
 
 const driverUpgradeHelpLink = MetabaseSettings.docsUrl(
   "administration-guide/01-managing-databases",
 );
 
-function getSupersedesWarningContent(newDriver, supersedesDriver) {
+function getSupersedesWarningContent(
+  newDriver,
+  supersedesDriver,
+  onChangeEngine,
+) {
   return (
     <div>
-      <p className="text-medium m0">
+      <WarningParagraph className="text-medium m0">
         {t`This is our new ${
           allEngines[newDriver]["driver-name"]
         } driver, which is faster and more reliable.`}
-      </p>
-      <p>{t`The old driver has been deprecated and will be removed in the next release. If you really
-      need to use it, you can select ${
-        allEngines[supersedesDriver]["driver-name"]
-      } now.`}</p>
+      </WarningParagraph>
+
+      <WarningParagraph className="text-medium m0" hasMargin={true}>
```

Comment:
1. You can just use `hasMargin` and it will be the same thing as if you pass `hasMargin={true}` directly
2. The margins are missing because thr `m0` CSS class overwrites it, so you can remove it
3. Please remove the `text-medium` class too, you can replace it with `color: ${color("text-medium")};` in the styled component definition

Reply:
Fixed all in next commit.